applications/nrf_desktop/README.rst

```diff
@@ -965,7 +965,7 @@ Sample mouse, keyboard or dongle (nrf52840dk_nrf52840)
       * The configuration uses the nRF52840 Development Kit.
       * The build types allow to build the application as mouse, keyboard or dongle.
       * Inputs are simulated based on the hardware button presses.
-      * The configuration with bootloader is available.
```

Comment:
nRF Desktop build types section is outdated

Reply:
Maybe we could even state there that by default B0 bootloader is enabled if such configuration fits in memory ?